src/dimension.h

```diff
@@ -167,6 +171,12 @@ extern void ts_dimension_info_validate(DimensionInfo *info);
 extern void ts_dimension_add_from_info(DimensionInfo *info);
 extern void ts_dimensions_rename_schema_name(char *oldname, char *newname);
 
+/* TODO:: Q:: this function should probably need to be refactored if we agree it should be exposed
+ */
+extern TSDLLEXPORT void dimension_update(FunctionCallInfo fcinfo, Oid table_relid, Name dimname,
```

Comment:
This used to be a static function in `dimension.c`. Its first `fcinfo` arg is used iff `Datum *interval` is not null and it is used to retrieve interval type. Should the interval type be passed directly instead?

The function is otherwise quite generic and makes it easy to update certain dimension table fields so may be useful to expose it.